Composer/packages/lib/code-editor/src/lg/modalityEditors/ModalityEditorContainer.tsx

```diff
@@ -105,10 +105,30 @@ export const ModalityEditorContainer: React.FC<Props> = ({
               {
                 confirmText: formatMessage('Confirm'),
                 onRenderContent: renderConfirmDialogContent,
+                choiceGroup:
+                  modalityType === 'Attachments'
+                    ? {
+                        selectedKey: 'remove',
+                        options: [
+                          {
+                            key: 'remove',
+                            text: formatMessage(
+                              'Yes, I’d like to remove the the content of this tab completely from the LG file.'
+                            ),
+                          },
+                          {
+                            key: 'keep',
+                            text: formatMessage(
+                              'No, I want to keep the content, just de-reference from this response.'
```

Comment:
de-reference could be confusing to some users.  How about this?

No, I want to keep the content in the LG file and only remove the reference from this response.

Reply:
@sangwoohaan Can you weigh in?